jwql/instrument_monitors/common_monitors/bad_pixel_monitor.py

```diff
@@ -894,16 +940,16 @@ def process(self, illuminated_raw_files, illuminated_slope_files, dark_raw_files
         output_file = '{}_{}_{}_bpm.fits'.format(self.instrument, self.aperture, query_string)
         output_file = os.path.join(self.output_dir, output_file)
 
-        logging.info("Calling bad_pixel_mask.bad_pixels")
-        logging.info("\tflat_slope_files are: {}".format(illuminated_slope_files))
-        logging.info("\tdead__search_type={}".format(dead_search_type))
-        logging.info("\tflat_mean_normalization_method={}".format(flat_mean_normalization_method))
-        logging.info("\tdead_flux_check_files are: {}".format(dead_flux_files))
-        logging.info("\tdark_slope_files are: {}".format(dark_slope_files))
-        logging.info("\tdark_uncal_files are: {}".format(dark_raw_files))
-        logging.info("\tdark_jump_files are: {}".format(dark_jump_files))
-        logging.info("\tdark_fitopt_files are: {}".format(dark_fitopt_files))
-        logging.info("\toutput_file={}".format(output_file))
+#         logging.info("Calling bad_pixel_mask.bad_pixels")
```

Comment:
Did you intend to leave this commented out? Can it be removed if yes?

Reply:
It can probably be removed. The reason I had it in place initially is that, whenever the bad pixel monitor crashed on making its file, I always wanted to do a re-run with the logging enabled, so I could see what files it thought it was calibrating, whether they all actually existed, and whether there were issues with any of them. It just seemed like a lot of extra text to add on occasions where the bad pixel file creation wasn't passing.

Thinking of it, I guess I'm happier if the log always prints out that diagnostic than if it never does, because the reference file creation functions can be a bit of a black box, and sometimes figuring out what's going wrong in the depth of them is difficult to trace.